styles.css

```diff
@@ -1 +1,48 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container{
+    display: grid;
+    grid-gap: 16px 16px;
+    background-color: rgb(252, 238, 240);
+    padding: 15px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 190px 100px auto 50px;
+    grid-template-areas:
+        "header aside-1 aside-1"
+        "main aside-1 aside-1"
+        "main aside-2 aside-3"
+        "main aside-2 ."
+        "footer footer footer"
+;
+}
+
+.grid-item{
+    background-color: rgb(157, 27, 53);
+    border: rgb(70, 12, 23);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
hier had die padding gemoeten om die ruimte rond de tekst in blok B te creëren, misschien was het per ongeluk in die DIV container beland.